src/changelog/sources/markdown/filter.go

```diff
@@ -0,0 +1,38 @@
+package markdown
+
+import (
+	"io"
+	"reflect"
+
+	"github.com/gomarkdown/markdown"
+	"github.com/gomarkdown/markdown/ast"
+)
+
+// FilterRenderer wraps a markdown.Renderer skips calling it for certain node types.
+// This allows to use the default markdown renderer with types it does not support without it panicking.
+type FilterRenderer struct {
+	filtered map[reflect.Type]bool
+	markdown.Renderer
+}
+
+// NewFilterRenderer returns a FilterRenderer given an underlying renderer and a list of nodes whose type will be ignored.
+func NewFilterRenderer(inner markdown.Renderer, ignore ...interface{}) FilterRenderer {
+	filtered := map[reflect.Type]bool{}
+	for _, iface := range ignore {
+		filtered[reflect.TypeOf(iface)] = true
+	}
+
+	return FilterRenderer{
+		filtered: filtered,
+		Renderer: inner,
+	}
+}
+
+// RenderNode implements markdown.Renderer with the additional filtering logic.
+func (fr FilterRenderer) RenderNode(w io.Writer, node ast.Node, entering bool) ast.WalkStatus {
+	if fr.filtered[reflect.TypeOf(node)] {
+		return ast.GoToNext
+	}
+
+	return fr.Renderer.RenderNode(w, node, entering)
+}
```

Comment:
Nit: I am aware this is already tested using the `md_test.go` but did you consider also adding a unit-test? (Even if it is a simple and potentially temporal wrapper)

Reply:
Added in https://github.com/newrelic/release-toolkit/pull/125/commits/7fb6b72db8ada112e715a6c681fc4e101f93c0ad

Reply:
👍